libs/backend/src/cast_vote_records/import.ts

```diff
@@ -256,12 +263,31 @@ export async function readCastVoteRecordExport(
       (batch) => batch['@id']
     )
   );
-  const castVoteRecords = iter(
+  const castVoteRecordIterator = iter(
     castVoteRecordGenerator(exportDirectoryPath, batchIds)
   );
 
   return ok({
     castVoteRecordExportMetadata,
-    castVoteRecords,
+    castVoteRecordIterator,
   });
 }
+
+/**
+ * Determines whether or not a cast vote record report was generated on a machine in test mode
+ */
+export function isTestReport(
```

Comment:
Another function that I needed to move to unblock removing an old file, in this case `legacy_import.ts`, in an upcoming PR